torch/_higher_order_ops/triton_kernel_wrap.py

```diff
@@ -1193,12 +1248,28 @@ def call_triton_kernel(
                 new_configs = copy.deepcopy(variable.kernel.configs)
                 for config in new_configs:
                     config.__dict__.update(special_kwargs)
-                new_kernel = autotune(configs=new_configs, key=[])(variable.kernel.fn)
+
+                prune_configs_by = {
+                    "perf_model": variable.kernel.perf_model,
+                    "early_config_prune": variable.kernel.early_config_prune,
+                    "configs_top_k": variable.kernel.configs_top_k,
+                }
+                new_kernel = autotune(
+                    configs=new_configs, prune_configs_by=prune_configs_by, key=[]
+                )(variable.kernel.fn)
```

Comment:
Can we extract this into a helper, as doing multiple times?

Reply:
I think we may not need this anymore, since prune_configs_by should be stripped from the kernel by there (we recurse and run call_triton_kernel with the new configs earlier)